cl/users/management/commands/cl_delete_old_messages.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime, timedelta
+
+from django.utils.timezone import make_aware, utc
+
+from cl.lib.command_utils import VerboseCommand
+from cl.users.models import EmailSent
+
+
+class Command(VerboseCommand):
+    """Command to delete stored messages older than the specified number of
+    days."""
+
+    help = "Delete stored messages older than the specified number of days."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--older-than-days",
+            type=int,
+            help="Delete messages older than this number of days.",
+            required=True,
+        )
+
+    def handle(self, *args, **options):
+        super(Command, self).handle(*args, **options)
+        self.options = options
+        messages = self.delete_old_messages(options["older_than_days"])
+        print(f"Messages deleted: {messages}")
+
+    def delete_old_messages(self, days):
+        """Delete stored messages older than the specified number of days."""
```

Comment:
I'd suggest moving this to a top-level function in this file. Then in your tests, you can just test this function and not call the full command, 

I realize doesn't test all your code here (while doing the full command does), but a smaller unit test should be faster and tidier.